march_hardware_builder/src/hardware_builder.cpp

```diff
@@ -135,9 +139,18 @@ std::unique_ptr<march::IMotionCube> HardwareBuilder::createIMotionCube(const YAM
   YAML::Node incremental_encoder_config = imc_config["incrementalEncoder"];
   YAML::Node absolute_encoder_config = imc_config["absoluteEncoder"];
   int slave_index = imc_config["slaveIndex"].as<int>();
+
+  std::ifstream imc_setup_data;
+  imc_setup_data.open(ros::package::getPath("march_hardware_builder").append("/config/" + urdf_joint->name + ".sw"));
+  std::cout << "jointname: " << urdf_joint->name << "path to file"
+            << ros::package::getPath("march_hardware_builder").append("/config/" + urdf_joint->name + ".sw")
+            << std::endl;
```

Comment:
Please remove these debug `cout` statements or replace them with the ROS log macros.
```suggestion
```